third_party/openvino/ngraph_c_api/src/ngraph_c_api.cpp

```diff
@@ -119,6 +119,48 @@ inline ngraph::op::PadType GetAutoPad(ngraph_auto_pad autoPad) {
   return auto_pad;
 }
 
+inline std::string GetGruActivation(ngraph_gru_activation activation) {
+  switch (activation) {
+    case Relu:
+      return "relu";
+    case Sigmoid:
+      return "sigmoid";
+    case Tanh:
+      return "tanh";
+    default:
+      assert(0);
+  }
+}
+
+inline std::vector<std::string> GetGruActivations(
+    ngraph_gru_activation reset_activation,
+    ngraph_gru_activation new_activation) {
+  std::vector<std::string> activations;
+  activations.push_back(GetGruActivation(reset_activation));
+  activations.push_back(GetGruActivation(new_activation));
+
+  return activations;
+}
+
+inline ngraph::op::RecurrentSequenceDirection get_recurrent_sequence_direction(
```

Comment:
Will it bring build warning of uninitialized varibles?

Reply:
Removed this function.